compiler/passes.cc

```diff
@@ -111,7 +111,11 @@ void RunDefaultPasses(Graph* graph, bool gen_backprop) {
             // specified computation order
             skip_scheduling = true;
             auto orders = GetComputationOrder(*graph, g_computation_order);
-            AddGradientNodesForTrainingWithOrders(graph, orders);
+            if (!AddGradientNodesForTrainingWithOrders(graph, orders)) {
+                std::cerr << "Computation order is not supported in this graph.\n";
```

Comment:
Doesn't `CHECK(false) << ...` won't work for you? Silently exiting with zero wouldn't be a great idea since it may get people overlook errors.

Reply:
It's a little difficult point. I wanted to suppress errors caused by this part when I run `python scripts/runtests.py --computation_order=dummy`. But as you say, we may easily overlook errors with this code.
Adding a "skipped" status in runtests.py may be convenient in this case, but I'm not sure if it is the best.

Reply:
OK. I guess we can figure out a better solution, but let's go ahead with this. Thanks for your explanation!